website/docs/about/overview.md

```diff
@@ -6,45 +6,63 @@ navigation:
   order: 101
 ---
 
-## What is a design system?
+## What is a design system
 
 A design system is a set of guidelines, standards, assets, and processes to help organizations design, build, and deploy products rapidly and consistently.
 
 Design systems are often equated with a UI kit or component library. In reality, an effective design system spans the entire lifecycle of a product, from ideation to iteration and everything in between.
 
-## Helios architecture
+## What is Helios
 
-While the scale of a design system largely depends on the needs of those consuming it, the fundamental structure and architecture remain largely the same across systems.
+**Helios** is the design system developed by the HashiCorp Design System Team. It’s commonly referred to as **HDS**.
```

Comment:
Not opinionated on this (and there's obviously other context on this page), but using the term 'developed' kind of reads to me like a engineering-focused effort. Is something agnostic better? Built? Created?
```suggestion
**Helios** is the design system built by the HashiCorp Design System Team. It’s commonly referred to as **HDS**.
```

Reply:
Ah, that's a good catch. I like this adjustment.